Nos/Controller/SearchController.swift

```diff
@@ -145,7 +171,18 @@ class SearchController: ObservableObject {
             }
         }
     }
-    
+
+    func startSearchTimer() {
+        isNotFindingResults = false
+        timer?.invalidate()
+
+        timer = Timer.scheduledTimer(withTimeInterval: 10, repeats: false) { _ in
```

Comment:
This is a bit of personal preference, but 10 seconds seems way too long to me. I doubt many users will stick around that long to wait for results. I would suggest cutting it to at least 5?

Reply:
I'll leave this per https://github.com/planetary-social/nos/pull/846#discussion_r1465336877